PanoptesNetClient/PanoptesNetClient/Config.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PanoptesNetClient
+{
+    public static class Config
```

Comment:
This class is also very javascripty. 

To make it more object oriented it may be a better idea to make it a class that takes an `env` parameter to start with and then sets its `Host`, `ClientAppId`, etc properties based on that initial constructor param.

Upsides to this are that you can create multiple config objects for different configurations. 

If you decide to take this route, it may be worth looking into [enums](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum) for that `env` variable.

Reply:
So the idea is that the static class takes a look at the env variable in startup and it serves the proper urls based on that?

Seems fine, I think all you should do is get rid of the static member variables and just leave the functions `API_HOSTS()`, `API_APPLICATION_IDS()`. If I am understanding static classes correctly, the class variables will only instantiate themselves once, so you would need to restart the app in order to change which env you are in.

However, if you just use the functions, you can keep the static class and also have the dynamic functionality of changing the env variables while the app is running. The functions are so small, I don't think it will be a performance concearn

Reply:
The approach here might be to use the .config extension and write a different .config file for production, staging, and development. There may also be a config manager that will determine which environment the app should run in and can be toggled through VS. In that case, a Config class (whether static or not) might not be necessary, although it would work fine. I'm going to explore this a bit.

Reply:
https://msdn.microsoft.com/en-us/library/system.configuration.configurationmanager(v=vs.110).aspx

Reply:
I found difficulties making this visible to a class library and am not sure it is possible. Perhaps I should use the `.settings` extension? I know `ConfigurationManager` is an extension of `System.Configuration`, but it wasn't becoming visible to those class libraries.

Reply:
It is possible to read from a config file in a class library, but once the class library is consumed in an app, it might want to read its settings from that app's app.config instead of its own

Reply:
The discussion here might be more useful: https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/index